roles/telemetry_autoscaling/tasks/verify_autoscaling.yml

```diff
@@ -1,4 +1,36 @@
 ---
+- name: Verify that custom resource HEAT is ready
+  ansible.builtin.command:
+    cmd: |
+      oc get heat heat -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}{"\n"}'
+  register: result
+  failed_when:
+    - result.stdout != "True"
+
+- name: Verify that custom resource CEILOMETER is ready
+  ansible.builtin.command:
+    cmd: |
+      oc get ceilometer ceilometer -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}{"\n"}'
+  register: result
+  failed_when:
+    - result.stdout != "True"
+
+- name: Verify that custom resource AUTOSCALING is ready
+  ansible.builtin.command:
+    cmd: |
+      oc get autoscaling autoscaling -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}{"\n"}'
+  register: result
+  failed_when:
+    - result.stdout != "True"
+
+- name: Verify that custom resource METRICSTORAGE is ready
+  ansible.builtin.command:
+    cmd: |
+      oc get metricstorage metric-storage -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}{"\n"}'
+  register: result
+  failed_when:
+    - result.stdout != "True"
+
```

Comment:
These look good.
Since they are specific to openstack running on openshift, they should be separated from the more generic checks that query the OpenStack APIs and are common to  tripleo and kubernetes-based deployments.

I suggest either add a playbook to ci/ and hook it into ci/run_playbooks.yml 
OR
put this into a separate task, and update roles/telemetry-autoscaling/tasks/main.yml to run this conditionally.

Reply:
Sure Emma.. I will modify them accordingly

Reply:
I have added the tests into a seperate task that are specific to openstack running on openshift with the condition "metrics_backend == "prometheus""

Reply:
recheck